src/ARKit/ARPlaneGeometry.cs

```diff
@@ -0,0 +1,67 @@
+//
+// ARPlaneGeometry.cs: Nicer code for ARPlaneGeometry
+//
+// Authors:
+//	Vincent Dondain  <vidondai@microsoft.com>
+//
+// Copyright 2018 Microsoft Inc. All rights reserved.
+//
+
+#if XAMCORE_2_0
+
+using System;
+using System.Runtime.InteropServices;
+using Vector2 = global::OpenTK.Vector2;
+using Vector3 = global::OpenTK.NVector3;
+
+namespace XamCore.ARKit {
+	public partial class ARPlaneGeometry {
+
+		// Using GetXXX methods so it's similar to the ARFaceGeometry methods.
```

Comment:
I think I prefer methods over props  when the operation is "expensive" like interfacing with unmanaged code. Just a personal preference :)